README.md

```diff
@@ -143,6 +144,27 @@ Or, in case of just a `from` with the default destination, you can also use a `{
 ]
 ```
 
+### `test`
+
+Defines a regExp to match some parts of the file path.
+These capture groups can be reused in the name property using `[N]` placeholder.
+Note that `[0]` will be replaced by the entire tested string,
```

Comment:
```

> ℹ️ Note
```

`string` => `{String}`

Reply:
Went with "entire relative path of the file" while I was touching it since that seemed easier to understand than what the "tested string" was.